README.md

```diff
@@ -67,3 +67,10 @@ Provide clean library abstractions with clear and well exampled godocs.
 - Provide code generators to maintain common boilerplate that can't be addressed by interfaces
   - Driven off of `//+` comments
 - Provide bootstrapping commands to initialize new packages
+
+### Troubleshooting
+
+If you have what looks like a bug, or you would like to make a feature request, please use the [Github issue tracking system.](https://github.com/kubernetes-sigs/kubebuilder/issues)
```

Comment:
We can add a subtitle `bugs and feature requests` here.

Reply:
Done